### PR TITLE
feat: move icons into sharepoint

### LINKED ISSUE #83: Move icons into word document
Move icons, that are embedded in the content – like the social media icons in the footer – from the icons folder in the code, to the word document.
This can be achieved by copying the svg file and pasting it in the Word file.
-> Please note that this might only work in the desktop version of word.

### PR DESCRIPTION
**Pull request by dzehnder**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #83 

Test URLs:
- Before: https://main--mammotome--hlxsites.hlx.page/en/
- After: https://83-move-icons--mammotome--hlxsites.hlx.page/en/

### CODE REVIEW

**Review by Acidham — CHANGES_REQUESTED** [review]
@dzehnder you also moved logo.svg, but there are two different flavors of the Mammotome logo, the wide version and the small version. To be consistent, I would expect to upload/change the small version as well?

File: blocks/header/header.js
Comment (Acidham) [inline blocks/header/header.js:1]: I would call it `logo-light` or `logo-small` and the logic would be that `:logo:` in word doc will be extended with `-light` resp. `-small`

Comment (dzehnder) [reply blocks/header/header.js:1]: The only place where it is used is in combination with custom code, i.e when the user scrolls down. 
I don't think it is the right approach to make the author responsible for that as it is more of a design decition.

**Review by solaris007 — APPROVED** [review]

**Comment by solaris007** [comment]
@dzehnder anything else required?

**Pull request merged**